templates/partials/committee-filter.html

```diff
@@ -8,5 +8,6 @@
 {% include 'partials/filters/election-years.html' %}
 {% include 'partials/filters/parties.html' %}
 {{ states.field('state') }}
+{{ text.field('treasurer', 'Treasurer name') }}
```

Comment:
Can we just move this to after "Name"? It's the second column, so I think it makes sense there.